src/main/java/NewMenteeServlet.java

```diff
@@ -42,6 +43,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             .build();
     datastore.put(menteeEntity);
 
-    response.sendRedirect("/index.html");
+    //response.sendRedirect("/matchingPage.html");
```

Comment:
It looks like this code was moved to the area above, so please delete. We want to try to keep the code clean and not to merge code to the master branch unless it's ready, so other people won't be confused by random commented out code.